src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Circle - radius:" + radius + "and the area is: " + calculateArea());
```

Comment:
The draw method should also include the color of the circle in the output to fully meet the requirement of printing figure information. Also add space after `radius:` and before `and the area is: ` for better readability.